mediaflow_proxy/extractors/dlhd.py

```
@@ -1,588 +1,323 @@
+                    logger.info(f"Constructed stream URL: {stream_url}")
+            
+            if not stream_url:
+                # Fallback: cerca qualsiasi URL che sembri uno stream
+                url_pattern = r'https?://[^\s"\'<>]+\.m3u8[^\s"\'<>]*'
+                matches = re.findall(url_pattern, iframe_content)
+                if matches:
+                    stream_url = matches[0]
+                    logger.info(f"Found fallback stream URL: {stream_url}")
+            
+            if not stream_url:
+                raise ExtractorError(f"Could not find stream URL in lovecdn.ru iframe")
```

Ruff (0.14.5)

Abstract `raise` to an inner function

(TRY301)